core/src/test/java/org/elasticsearch/search/suggest/AbstractSuggestionBuilderTestCase.java

```diff
@@ -135,13 +135,17 @@ public void testSerialization() throws IOException {
      */
     protected SB randomTestBuilder() {
         SB randomSuggestion = randomSuggestionBuilder();
-        maybeSet(randomSuggestion::text, randomAsciiOfLengthBetween(2, 20));
+        setCommonPropertiesOnRandomBuilder(randomSuggestion);
```

Comment:
If we call this in every test implementation we don't need to call it again here, wdyt?